### PR TITLE
added MusicCard component and responsive grid layout

### PR DESCRIPTION
**Pull request by tousifws**
### Ticket Link
---------------------------------------------------

### Related Links
---------------------------------------------------

### Description
---------------------------------------------------

- added MusicCard component 
- responsive grid layout

### Steps to Reproduce / Test
---------------------------------------------------

### GIF's
---------------------------------------------------
itunes_grid

itunes_resp

### CODE REVIEW

File: app/components/MusicCard/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * Card
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Card, Avatar } from 'antd';
+// import { FormattedMessage } from 'react-intl';
```
Comment (alichherawalla) [inline app/components/MusicCard/index.js]: remove commented out code

File: app/components/MusicCard/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import T from '@components/T';
+
+const MusicContent = styled.div`
+  display: flex;
+  align-items: center;
+  height: 120px;
```
Comment (alichherawalla) [inline app/components/MusicCard/index.js]: use em instead of px

File: app/containers/MusicContainer/index.js [hunk 3]
```diff
@@ -96,32 +92,32 @@ export function MusicContainer({
...
-                  <Avatar src={item.artworkUrl100} shape="square" size={100} />
-                  <div style={{ paddingLeft: '0.6em' }}>
-                    <T id="music_title" values={{ title: item.trackName }} />
-                    <T id="artist_name" values={{ artistName: item.artistName }} />
-                    <T id="music_collection_name" values={{ collectionName: item.collectionName }} />
-                  </div>
-                </MusicContent>
-              </CustomCard>
-            ))}
+            <Row justify="center">
```
Comment (alichherawalla) [inline app/containers/MusicContainer/index.js:107]: this is creating new columns for every track

Comment (tousifws) [reply app/containers/MusicContainer/index.js:107]: Yes. It's looking fine

**Pull request merged**